packages/react-components/src/atoms/__tests__/Link.test.tsx

```diff
@@ -199,3 +199,56 @@ describe('when button-styled', () => {
     expect(getByText('text').closest('a')).not.toHaveAttribute('href');
   });
 });
+
+describe('for elipsed links', () => {
+  // !! Unfortunately jest does not respect ellipsed texts (overflow: hidden)
+  // https://github.com/testing-library/react-testing-library/issues/751#issuecomment-691069646
+  // so we cannot test if the text is shown breaked as it should
+  // but we can test other properties and see it working in the StoryBook
```

Comment:
We do have playwright to test things like this

Reply:
For now we don't have any page using this Link component. I guess the components using it are going to be implemented in the future and then we'll be able to introduce the `e2e` tests. But anyways I can remove these comments.

Reply:
I've talked with João and he has provided me a page where we can use it for now. Will do it.